python/katana/local/_graph.pyx

```diff
@@ -481,6 +531,182 @@ cdef class GraphBase:
+        `entity_type_id` for nodes
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        return handle_result_TypeNameSet(manager.EntityTypeToTypeNameSet(entity_type_id))
+
+    def edge_entity_type_to_type_name_set(self, entity_type_id: int):
+        """
+        Return a set of strings representing the names of the types for the type ID
+        `entity_type_id` for edges
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        return handle_result_TypeNameSet(manager.EntityTypeToTypeNameSet(entity_type_id))
+
+    def get_atomic_node_type_name(self, entity_type_id: int):
+        """
+        Return the type name if `entity_type_id` is a valid node entity type ID. Otherwise,
+        return None
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        r =  manager.GetAtomicTypeName(entity_type_id)
+        if r:
+            return r.value()
+        return None
+
+    def get_atomic_edge_type_name(self, entity_type_id: int):
+        """
+        Return the type name if `entity_type_id` is a valid edge entity type ID. Otherwise,
+        return None
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        r =  manager.GetAtomicTypeName(entity_type_id)
+        if r:
+            return r.value()
+        return None
+
+    def get_atomic_node_entity_type_ids(self):
+        """
+        Return a list of all node entity type IDs
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        return manager.GetAtomicEntityTypeIDs()
+
+    def get_atomic_edge_entity_type_ids(self):
+        """
+        Return a list of all edge entity type IDs
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        return manager.GetAtomicEntityTypeIDs()
+
+    def node_is_subtype_of(self, sub_type: int, super_type: int):
+        """
+        Return True iff the type `sub_type` is a subtype of the type `super_type` for
+        nodes
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        return manager.IsSubtypeOf(sub_type, super_type)
+
+    def edge_is_subtype_of(self, sub_type: int, super_type: int):
+        """
+        Return True iff the type `sub_type` is a subtype of the type `super_type` for
+        edges 
+        """
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetEdgeTypeManager()
+        return manager.IsSubtypeOf(sub_type, super_type)
+
+    def get_node_entity_type_id_to_atomic_type_name_map(self):
+        """
+        Return the map from node entity type IDs to type names
```

Comment:
The main map type in Python is called `dict`. I assume that's what you are getting here.